app/main.py

```diff
@@ -2,0 +1,19 @@
+def copy_file(command: str) -> None:
+    command_line = command.split()
+    command_name, source_file, copy_file_name = command_line
+    is_valid_file_names = source_file != copy_file_name
+    is_valid_command = command_name == "cp"
+    is_valid_count_of_parameters = len(command_line) == 3
+    if all(is_valid_command,
+           is_valid_file_names,
+           is_valid_count_of_parameters
+           ):
+        try:
+            with (open(source_file, "r") as f_in,
+                  open(copy_file_name, "w") as f_out
+                  ):
+                f_out.write(f_in.read())
+        except (FileNotFoundError, FileExistsError):
+            print(f"Source file {source_file} does not exist.")
```

Comment:
Catching `FileExistsError` here is not appropriate. This error is raised when trying to create a file that already exists, which is not applicable in this context since `open(copy_file_name, 'w')` will overwrite the file if it exists. You should only catch `FileNotFoundError` here.